packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html

```diff
@@ -1,247 +1,179 @@
-                                type="button"
-                                data-ng-if="!singleService()"
-                                data-ng-repeat="ip in alerts.arp"
-                                data-ng-click="navigateToService(ip.ip.routedTo.serviceName)"
-                                data-ng-bind="ip.ip.routedTo.serviceName"
-                            ></button>
-                        </p>
-
-                        <p data-ng-if="alerts.mitigation.length">
-                            <span
-                                class="d-block"
-                                data-ng-pluralize
-                                data-count="alerts.mitigation.length"
-                                data-when="{
-                                        '0': ('ip_alerts_mitigation' | translate),
-                                        'one': ('ip_alerts_mitigation_1' | translate),
-                                        'other': ('ip_alerts_mitigation_other' | translate: { count: {} })
-                                    }"
-                            >
-                            </span>
-                            <button
-                                class="btn btn-link"
-                                type="button"
-                                data-ng-if="!singleService()"
-                                data-ng-repeat="ip in alerts.mitigation"
-                                data-ng-click="navigateToService(ip.ip.routedTo.serviceName)"
-                                data-ng-bind="ip.ip.routedTo.serviceName"
-                            ></button>
-                        </p>
-                    </div>
+                    </span>
+                    <span
+                        data-ng-if="singleService()"
+                        data-ng-pluralize
+                        data-count="alertsCount()"
+                        data-when="{
+                                '0': ('ip_alerts_service_none' | translate) + ' :',
+                                'one': ('ip_alerts_service_1' | translate) + ' :',
+                                'other': ('ip_alerts_service_other' | translate: { count: {} }) + ' :'
+                            }"
+                    >
+                    </span>
+                </p>
 
-                    <div
-                        class="alert alert-success"
-                        data-ng-if="!alertsCount()"
-                        role="alert"
+                <p data-ng-if="alerts.spam.length">
+                    <span
+                        class="d-block"
+                        data-ng-pluralize
+                        data-count="alerts.spam.length"
+                        data-when="{
+                                '0': ('ip_alerts_spam' | translate),
+                                'one': ('ip_alerts_spam_1' | translate),
+                                'other': ('ip_alerts_spam_other' | translate: { count: {} })
+                            }"
```

Comment:
**nitpick:** Can we replace `<span>` + `.d-block` with the block level elements?